src/components/rockets/RocketList.js

```diff
@@ -24,6 +24,7 @@ const RocketList = () => {
             rocketName={rocket.rocket_name}
             description={rocket.description}
             flickrImages={rocket.flickr_images}
+            reserved={rocket.reserved}
           />
         ))}
       </ul>
```

Comment:
* Issue can be found in line 11 to 15 *

Reserved rockets state is not being saved properly after the directory changes. I think it's because you are dispatching loadRockets within useEffect every time the rockets component is rendered. That resets the state and doesn't preserve the changes through the use of the app. Kindly find a way to fetch the rockets one time so that the state works properly 🤓